### ISSUE TITLE
build reactome.jar from CuratorTool

### ISSUE THREAD

**Issue by SolomonShorser-OICR** [reactome/container#8]
RESTfulAPI depends on a file "reactome.jar" which is built from the code of the CuratorTool repo.
There is an ant task to build the jar, but there is no task to compile the source code first.
A new ant task is needed to compile to code before the jar can be built.

**Comment by SolomonShorser-OICR** [comment]
This will need to be done before https://github.com/reactome/container/issues/7

**Comment by AyushyaChitransh** [comment]
I am working on this issue! This one is at first priority.

**Comment by SolomonShorser-OICR** [comment]
I think you're right: using issues to track work is better than those little "notes" in the Project page.

**Comment by AyushyaChitransh** [comment]
Trying to resolve some errors, which were mostly related to including packages:
```
error: package org.apache.batik.svggen does not exist
error: package org.jdom does not exist
error: package org.junit does not exist
error: package org.apache.batik.ext.swing does not exist
...
```
Probably I must have missed some  library or not specified classpath. Looking into it.

**Comment by AyushyaChitransh** [comment]
Previously only base directory was specified in classpath, but after specifying in buildfile to look recursively, all those errors were gone. 
There still are some errors regarding `unit tests`
```
error: package org.junit does not exist
    [javac] import org.junit.Test;
```

**Comment by AyushyaChitransh** [comment]
I think the `junit.jar` file in `CuratorTool/lib/junit` is obsolete. I downloaded new jar from [maven][1], I could build reactome.jar successfully. Should I make a PR to update `junit.jar` there also?

[1]: https://mvnrepository.com/artifact/junit/junit/4.12

**Comment by AyushyaChitransh** [comment]
Fixed with commit bcc7e2082c303ead893f01f70e5e248e79492075

**Comment by AyushyaChitransh** [comment]
The jar file will no longer be needed when [CuratorTools/lib/junit/][1] is updated. For now, [junit.jar][2] has been provided externally by downloading from [maven repository][3].

[1]: https://github.com/reactome/CuratorTool/tree/master/lib/junit
[2]: http://central.maven.org/maven2/junit/junit/4.12/junit-4.12.jar
[3]: https://mvnrepository.com/artifact/junit/junit/4.12
